content/introduction.html

```diff
@@ -1,15 +1,18 @@
 <h1> About the program </h1>
 <p class="full">
-  The University of Waterloo’s Software Engineering (SE) is an interdisciplinary
+  The Software Engineering (SE) at the University of Waterloo is an interdisciplinary
```

Comment:
```suggestion
  Software Engineering (SE) at the University of Waterloo is an interdisciplinary
```

or "The Software Engineering program"